react/components/CostCentersAutocomplete.tsx

```diff
@@ -59,32 +66,42 @@ const CostCenterAutocomplete = ({ onChange, organizationId }: Props) => {
         search: debouncedSearchTerm,
       })
     } else if (debouncedSearchTerm === '') {
-      onClear()
+      refetch({
```

Comment:
No need to replicate te refetch here, right? 

Can just send the onChange event on the else.